jobs/tests/test_models.py

```diff
@@ -127,3 +128,13 @@ def test_get_previous_approved(self):
     def test_region_optional(self):
         job = self.create_job(region=None)
 
+    def test_email_on_submit(self):
+        job = self.create_job()
+        mail.outbox = []
```

Comment:
This line can be removed. Django's TestCase class already does this: https://docs.djangoproject.com/en/dev/topics/testing/tools/#emptying-the-test-outbox